components/script/dom/windowproxy.rs

```diff
@@ -248,17 +262,69 @@ impl WindowProxy {
             ScriptThread::process_attach_layout(new_layout_info, document.origin().clone());
             let msg = EmbedderMsg::BrowserCreated(new_top_level_browsing_context_id);
             window.send_to_embedder(msg);
+            // TODO: if noopener is false, copy the sessionStorage storage area of the creator origin.
```

Comment:
File an issue for that.

Reply:
Ok https://github.com/servo/servo/issues/21291